themes/docs-new/layouts/shortcodes/ctl_chef_server_restore_options.md

```diff
@@ -9,4 +9,16 @@ This subcommand has the following options:
 
 :   Use to specify that the path to an empty directory to be used during
     the restore process. This directory must have enough disk space to
-    expand all data in the backup archive.
\ No newline at end of file
+    expand all data in the backup archive.
+
+`--pg-options`
+
+:   Use to specify additional options to pass to PostgreSQL during restore. See the [PostgreSQL documentation](https://www.postgresql.org/docs/13/runtime-config.html) for more information.
```

Comment:
```suggestion
```suggestion
:   Use to specify and pass additional options PostgreSQL during backups. See the [PostgreSQL documentation](https://www.postgresql.org/docs/13/runtime-config.html) for more information.
```
```